VitaminJProcessing.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  4 02:05:08 2024
+
+@author: Anupama Rajendra
+"""
+import csv
+import numpy as np
+import pandas as pd
+import openmc
+
+#Obtaining results from statepoint file:
+with openmc.StatePoint('statepoint.10.h5') as sp:
+    fl = sp.get_tally(name="Flux spectrum")
+    # Get the neutron energies from the energy filter
+    energy_filter_fl = fl.filters[0]
+    energies_fl = energy_filter_fl.bins[:, 0]
```

Comment:
Could probably combine this into one line.  Are these upper bounds or lower bounds (I assume upper).
```suggestion
    upper_ebounds = flux_tally.filters[0].bins[:, 0]
```